src/modules/navigator/mission.cpp

```diff
@@ -287,7 +371,9 @@ Mission::find_offboard_land_start()
 			break;
 		}
 
-		if (missionitem.nav_cmd == NAV_CMD_DO_LAND_START) {
+		if ((missionitem.nav_cmd == NAV_CMD_DO_LAND_START) ||
```

Comment:
Won't this change the RTL planned mission landing behaviour?

Reply:
I just checked it and using QGC one cannot mess it up because a land start waypoint is required. However, I could also add for the `NAV_CMD_LAND` waypoint the additional requirement that the vehicle needs to be a vtol.